src/test/java/posters/pageobjects/components/AddressForm.java

```diff
@@ -0,0 +1,44 @@
+package posters.pageobjects.components;
+
+import static com.codeborne.selenide.Condition.exactText;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import com.xceptance.neodymium.util.Neodymium;
+
+import posters.dataobjects.Address;
+
+public class AddressForm extends AbstractComponent
+{
+    private final String title;
+
+    public AddressForm(String title)
+    {
+        this.title = title;
+    }
+
+    public void isComponentAvailable()
+    {
+        $(".header-container>h2").shouldHave(exactText(title));
```

Comment:
@oomelianchuk Please see if there is a surrounding form or an element that os better to locate than the headline.

Reply:
replaced with assertion that "#fullName" is visible, which previously was placed in `fillAddressForm` method

Reply:
@oomelianchuk not sure if this does the trick, if I remember correctly #fullName is used at more places than the address form. Please check this.